develop/api_metadata.rst

```diff
@@ -327,6 +361,37 @@ Here's an example of how ``create_distributed_function()`` adds entries to the
     distribution_argument_index |
     colocationid                |
 
+.. _citus_tables:
+
+Citus tables view
+~~~~~~~~~~~~~~~~~
+
+The citus_tables view shows a summary of all tables managed by Citus (distributed and reference tables). The view combines information from Citus metadata tables for an easy, human-readable overview of these table properties:
+
+* :ref:`Table type <table_types>`
+* :ref:`Distribution column <dist_column>`
+* :ref:`Colocation group <colocation_groups>` id
+* Human-readable size
+* Shard count
+* Owner (database user)
+* Access method (heap or :ref:`columnar <columnar>`)
+
+Here's an example:
+
+.. code-block:: sql
+
+  SELECT * FROM citus_tables;
+
+::
+
+  ┌──────────┬──────────────────┬─────────────────────┬───────────────┬─────────┬─────────────┬───────┬───────────────┐
+  │   Name   │ Citus Table Type │ Distribution Column │ Colocation ID │  Size   │ Shard Count │ Owner │ Access Method │
```

Comment:
The names will also need to be updated.

Reply:
Oops, I was looking at an old version of the code, before Marco's PR.

Reply:
Where in `src/backend/distributed/build/sql/citus--9.5-1--10.0-1.sql` does it have the old names?

Reply:
Sorry I think I was mistaken. Not sure what I had open in my editor, must have been src/backend/distributed/sql/udfs/citus_tables/10.0-1.sql prior to commit 
https://github.com/citusdata/citus/commit/03328e9679e08685f8e24f77a577b17a4b9176b9